### ISSUE TITLE
Render Library Layouts in Theme Hook

### ISSUE THREAD

**Issue by raydale** [pojome/elementor#417]
Hi, the new library/layouts functionality is great so far!

Is it possible to render a library item inside of a theme hook? For example in Beaver Builder I can render a template (similar to library items in Elementor)  inside of my theme footer with a method: https://www.wpbeaverbuilder.com/knowledge-base/rendering-layouts-with-php.

Is there an API to do this in Elementor? If not could this be a request for one?

**Comment by rawzone** [comment]
This would be an awesome feature!

Could be very usefull for building archive pages etc. and keeping the "look" of the page by including for instance header and footer template-parts in the archive.php file.

And combined with the [Pods plugin](http://pods.io/) this would a match made in heaven if you not up for programming PHP and still want to build advanced (non onepage) setups with Elementor and Pods.

**Comment by Basilakis** [comment]
@arielk Any update here?

**Comment by arielk** [comment]
@Basilakis Unfortunately not yet, but we are working on it these days, I hope it will be released in the coming versions.

**Comment by amirhoseinsalemi** [comment]
any news on modifying search result page with elementor?

**Comment by arielk** [comment]
We added this feature to E-Pro 2.0.0

Thanks for your feedback!

**Comment by TerenceMilbourn** [comment]
I am as (or even more) exited about this release as I was about Thesis, back in the day.

**Comment by eatbuildplay** [comment]
Is there documentation of the feature somewhere, what classes/methods we could use to "load/fetch" a template and then in our use case, strip the content and replace it in a repeater?

**Comment by oooh-boi** [comment]
It's fun to see how EVERY topic on the subject of the relatively poor functionality (API-wise) of Templates Library (specifically a Local library) ends up here with the same sentence:
 "I merged this issue with https://github.com/elementor/elementor/issues/417 , which are in the same subject so we could deal with it more efficiently."
 ...even though these topics don't have anything in common, especially this one. 
 
What I don't understand is how the closing comment resolves all of the inquiries: 
"We added this feature to E-Pro 2.0.0"

I mean you have merged multiple different and potentially helpful topics into a non-related one which is now closed. Very misleading and inefficient. 
This comment is a result of frustration when searching for helpful information as a developer, I didn't mean to be offensive. Cheers!
